### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by bastionJersey** [balloob/pywemo#8]
Sometimes I get this error, and the device (a Wemo switch) doesn't toggle: 

`Traceback (most recent call last):
  File "/Users/adam/Dropbox/computer/wemotoggle.py", line 4, in <module>
    devices[0].toggle()
IndexError: list index out of range`

I call a simple shell script from within Applescript on sleep and wake of my Mac Mini, to turn the monitor on and off. This is the script I use: 

```
#!/usr/bin/env python
import pywemo
devices = pywemo.discover_devices()
devices[0].toggle()
```

At a guess there's no 0th element because it's not finding my device. It works a fair amount of the time, though. Is this a bug in pywemo, a problem with the Wemo switch, or do I need to add a loop to my shell script to keep calling discover_devices() until it finds one, then call toggle().

**Comment by balloob** [comment]
Sometimes wemos can be too slow to respond. You can pass in an integer as
first param to discover_devices to increase time it keeps looking.

**Comment by bastionJersey** [comment]
That doesn't seem to make any difference; am I doing it wrong? (I tried small numbers like 10 to start with.)

``devices = pywemo.discover_devices(1000000)``

In fact, it doesn't discover my device at all since yesterday. Running from the python interpreter in the terminal. 

```
>>> devices = pywemo.discover_devices()
>>> print devices
[]
>>> 
```

I also have ouimeaux installed and 'wemo list' finds the switch. 

```
wemo list
Switch: WeMo Switch
```

I've tried unplugging the switch and plugging it back in, but that doesn't help. Blasted switch is a mystery!
